src/components/DatePickerSettings/DatePickerSettings.tsx

```diff
@@ -26,6 +27,8 @@ import {
 } from '../../modules/datePicker';
 
 import { TOOLTIP_MOTION } from '../../constants';
+import { Field } from '../FilterSettings/FilterSettings.styles';
+import { setName } from '../../modules/datePicker/actions';
```

Comment:
I suggest to export `datePickerActions` from datePicker module and use it here - as part of our long goal refactoring.